Updated inactive anchor links and changed old url paths in admin-onboarding-tasks.rst

krutarththakkar opened the pull request:
Updated Old Url Paths with Anchors for better navigation

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
-->

#### Summary
<!--
A description of what this pull request does.
-->

#### Ticket Link
<!--
If this pull request addresses a Help Wanted ticket, please link the relevant GitHub issue, e.g.

  Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX

Otherwise, link the JIRA ticket.
-->

mattermod commented:
Hello @krutarththakkar,

Thanks for your pull request! A Core Committer will review your pull request soon. For code contributions, you can learn more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/).

krutarththakkar commented:
The old links on the site redirect to new paths, hence the anchor tags dont work out.
Hence, I updated the links to the new paths, and added relevant anchors so the user can navigate to the desired information like before, on clicking the links.

justinegeffen commented on source/getting-started/admin-onboarding-tasks.rst:
```diff
@@ -84,7 +84,7 @@ You can enable plugins and integrations to connect your team's workflows and too
+To enable and manage plugins, go to **System Console > Plugins**. Next, install plugins with a single click from the **Main Menu > Plugin Marketplace**. See the `Plugin Marketplace  <https://developers.mattermost.com/integrate/admin-guide/admin-plugins-beta/#plugin-marketplace>`__ documentation for details.
 
 Consider installing and enabling the following productivity plugins for your users:   
   - Enable audio/video calling and screen sharing with `Jitsi <https://integrations.mattermost.com/?s=Jitsi%20Plugin>`__ or `Zoom <https://integrations.mattermost.com/?s=Zoom%20Plugin>`__.
```
```suggestion
  - Enable audio/video calling and screen sharing with `Jitsi <https://mattermost.com/blog/mattermost-and-jitsi/>`__ or `Zoom <https://mattermost.com/marketplace/zoom-plugin/>`__.
```

justinegeffen reviewed (commented):
Just one suggestion otherwise this looks good to me, thank you! :)

krutarththakkar commented:
Thanks, committed your suggestion and one new commit,
Added missing anchors where needed.
Please review final changes :)

justinegeffen commented on source/getting-started/admin-onboarding-tasks.rst:
```diff
@@ -87,7 +87,7 @@ You can enable plugins and integrations to connect your team's workflows and too
 Consider installing and enabling the following productivity plugins for your users:   
-  - Enable audio/video calling and screen sharing with `Jitsi <https://integrations.mattermost.com/?s=Jitsi%20Plugin>`__ or `Zoom <https://integrations.mattermost.com/?s=Zoom%20Plugin>`__.
+  - Enable audio/video calling and screen sharing with `Jitsi <https://mattermost.com/blog/mattermost-and-jitsi/>`__ or `Zoom <https://mattermost.com/marketplace/zoom-plugin/>`__.
   - Create polls with `Matterpoll <https://integrations.mattermost.com/?s=Matterpoll%20Plugin>`__.
```
Curious as to why the following three links weren't changed along with the previous two?

justinegeffen replied:
@krutarththakkar, would you be open to updating all the links here so they have the correct URLs (i.e., do not include the search string but are the direct URL). Thank you. :)

krutarththakkar replied:
Ok, Am on it!

justinegeffen commented on source/about/corporate-directory-integration.rst:
```diff
@@ -19,7 +19,7 @@ Security features for authentication
 A core set of features is available with all authentication options to help increase security:
 
-- Ability to `set session length <https://docs.mattermost.com/configure/configuration-settings.html>`__ to define how long a user can use Mattermost before needing to re-enter credentials.
+- Ability to `set session length <https://docs.mattermost.com/configure/configuration-settings.html#id33>`__ to define how long a user can use Mattermost before needing to re-enter credentials.
```
This doesn't seem like a valid URL. Please double-check and provide the correct one. Thank you! :)

krutarththakkar commented:
How about now?
I guess most things are resolved!

justinegeffen commented:
Thanks, @krutarththakkar. I have provided a second review and comment about the four Marketplace links. Let me know if you have any questions! :)

krutarththakkar commented:
Updated everything,
Awaiting final review.

justinegeffen reviewed (approved):
This is a great PR! Thank you for all the effort you put into it. :)
